Source/WebKitLegacy/mac/WebView/WebUIDelegatePrivate.h

```diff
@@ -106,6 +106,7 @@ enum {
     WebMenuItemTagAddHighlightToNewQuickNote,
     WebMenuItemTagRevealImage,
     WebMenuItemTagTranslate,
+    WebMenuItemTagSwapCharacters,
```

Comment:
Nit - this is also a (very mild) bincompat risk. We should add the new enums at the end.